cmd/vic-machine/common/target.go

```diff
@@ -56,6 +58,16 @@ func (t *Target) TargetFlags() []cli.Flag {
 			Value: flags.NewOptionalString(&t.Password),
 			Usage: "ESX or vCenter password",
 		},
+		cli.BoolTFlag{ // TODO: s/BoolTFlag/BoolFlag/ - should be false by default
+			Name:        "insecure",
```

Comment:
Yes, specifying thumbprint implies insecure=false. I'm not seeing a method named processCertificates anywhere tho?